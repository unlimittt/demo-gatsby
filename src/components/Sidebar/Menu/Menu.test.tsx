import React from "react"
import Menu from "./Menu"
import Enzyme, { shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"

Enzyme.configure({ adapter: new Adapter() })

describe("Menu", () => {
  const props = {
    menu: [
      {
        label: "Item 0",
        path: "/#0/",
      },
      {
        label: "Item 1",
        path: "/#1/",
      },
    ],
  }

  it("renders correctly", () => {
    const wrapper = shallow(<Menu {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
