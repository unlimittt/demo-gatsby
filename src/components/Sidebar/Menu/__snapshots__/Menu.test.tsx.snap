// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<nav
  className="menu"
>
  <ul
    className="menu__list"
  >
    <li
      className="menu__list-item"
      key="/#0/"
    >
      <mockConstructor
        activeClassName="menu__list-item-link--active"
        className="menu__list-item-link"
        to="/#0/"
      >
        Item 0
      </mockConstructor>
    </li>
    <li
      className="menu__list-item"
      key="/#1/"
    >
      <mockConstructor
        activeClassName="menu__list-item-link--active"
        className="menu__list-item-link"
        to="/#1/"
      >
        Item 1
      </mockConstructor>
    </li>
  </ul>
</nav>
`;
